jekyll/_cci2/workflows.md

```diff
@@ -398,14 +383,30 @@ workflows:
               only: /^config-test.*/
             branches:
               ignore: /.*/
-
 ```
 
-**Note:** Webhook payloads from GitHub [are capped at 5MB](https://developer.github.com/webhooks/#payloads) and [for some events](https://developer.github.com/v3/activity/events/types/#createevent) a maximum of 3 tags. This means that if you push a lot of tags we may not receive all of them.
+**Note:**
+Webhook payloads from GitHub [are capped at 5MB](https://developer.github.com/webhooks/#payloads)
+and [for some events](https://developer.github.com/v3/activity/events/types/#createevent) a maximum of 3 tags.
+If you push several tags at once,
+CircleCI may not receive all of them.
 
 ### Regular Expression Support
 
-CircleCI branch and tag filters support the Java variant of regex pattern matching, see the [java.util.regex documentation](https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html) for details. 
+CircleCI branch and tag filters support the Java variant of regex pattern matching.
+When writing filters,
+CircleCI matches exact regular expressions.
```

Comment:
I'm not sure if it's documented elsewhere, but when regexp is used to filter tags/branches, we require the entire tag/branch to match.  That is to say:

- `/^...` and `/...$/` anchors aren't meaningful, as the entire target must match.  I noticed we use it in a lot of places.

Reply:
Oh we do mention the entire string must match here:

https://github.com/circleci/circleci-docs/pull/2532/files#diff-88159e45e163dbf450db5eccfcf89ae7R1015

But should probably mention that again in this section.